modAionImpl/src/org/aion/zero/impl/AionBlockchainImpl.java

```diff
@@ -538,9 +538,10 @@ public synchronized void compactState() {
 
     // TEMPORARY: here to support the ConsensusTest
     public Pair<ImportResult, AionBlockSummary> tryToConnectAndFetchSummary(
-            AionBlock block, long currTimeSeconds) {
+            AionBlock block, long currTimeSeconds, boolean doExistCheck) {
```

Comment:
I think you're referring to the other boolean constant that disabled the time check. The public `tryToConnectAndFetchSummary` currently appears on all branches. This boolean flag is needed here, otherwise the block will not get imported because it's already in the database.